### PR TITLE
REF: model family

### PR DESCRIPTION
**Pull request by UranusSeven**

### CODE REVIEW

**Review by RayJi01 — COMMENTED** [review]
LGTM

File: xinference/model/llm/pytorch/vicuna.py [hunk 1]
```diff
@@ -1,49 +0,0 @@
...
-            model_uid,
-            model_spec,
-            model_path,
-            system_prompt=self._system_prompt,
-            sep=self._sep,
-            user_name=self._user_name,
-            assistant_name=self._assistant_name,
-            pytorch_model_config=pytorch_model_config,
-        )
-        self._use_fast_tokenizer = False
```
Comment (pangyoki) [inline xinference/model/llm/pytorch/vicuna.py:49]: `_use_fast_tokenizer` property is missing

**Review by pangyoki — APPROVED** [review]
LGTM

**Review by pangyoki — APPROVED** [review]
LGTM

**Pull request merged**